docs/api/js/paymaster-utils.md

```diff
@@ -18,6 +18,12 @@ export const IPaymasterFlow = new ethers.utils.Interface(require('../../abi/IPay
 
 Returns encoded input for an approval-based paymaster.
 
+#### Inputs
+
+| Parameter           | Type                    | Description                     |
+| ------------------- | ----------------------- | ------------------------------- |
+| `paymasterInput`    | `BytesLike`             | The input data to the paymaster.|
```

Comment:
this is not a BytesLike

Reply:
@kailash-manasarovar, what could it be? Cause the [paymasterparams](https://era.zksync.io/docs/api/js/types.html#paymasterparams) page says it is.

Reply:
@kailash-manasarovar, what could it be? Cause the [paymasterparams](https://era.zksync.io/docs/api/js/types.html#paymasterparams) page says it is.

Reply:
check the code

Reply:
Okay, but if you check the [overrides](https://era.zksync.io/docs/api/js/features.html#overrides), you see this:

```
{
    overrides: {
        customData: {
            gasPerPubdata?: BigNumberish;
            factoryDeps?: BytesLike[];
            customSignature?: BytesLike;
            paymasterParams?: {
                paymaster: Address;
                paymasterInput: BytesLike;
            };
        }
    }
}

```
**paymasterInput** type is defined as `BytesLike`.

I'll confirm the type with a dev.